cmd/lakefs/cmd/setup.go

```diff
@@ -50,10 +50,7 @@ var setupCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		authService := auth.NewDBAuthService(
-			dbPool,
-			crypt.NewSecretStore(cfg.GetAuthEncryptionSecret()),
-			cfg.GetAuthCacheConfig())
+		authService := auth.NewDBAuthService(dbPool, crypt.NewSecretStore(cfg.GetAuthEncryptionSecret()), cfg.GetAuthCacheConfig(), logging.Default().WithField("service", "auth_service"))
```

Comment:
general note - usually I split these long initializations into multiple lines so it will be easy to diff and spot changes when a new services/parameter is added/removed.